doc/api/stream.md

```diff
@@ -1359,6 +1382,12 @@ The `writable._writev()` method is prefixed with an underscore because it is
 internal to the class that defines it, and should never be called directly by
 user programs.
 
+#### writable.\_destroy(err, callback)
+
+* `err` {Error} An error.
+* `callback` {Function} A callback function (optionally with an error
```

Comment:
A callback function that takes an optional error argument which is invoked when the writable is destroyed.